ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -167,14 +165,6 @@ Item {
         }
     }
 
-    Connections {
```

Comment:
I've fixed this connection in my merged PR, but your solution is more correct, we don't need to check `DisableAll` here. Just a note for you for easier rebase, simply take your code here 🙂